### PR TITLE
feat: cosmos 'Unstaking Confirm' modal

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

- add `<UnstakingConfirm />` modal 

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

#666

## Risk

More debt as we go with separate modals vs. one modal with memory routers (cc @reallybeard)

## Testing

- Open Cosmos Unstaking Modal
- Input some unstaking amount and click "Continue"

## Screenshots (if applicable)

image

### CODE REVIEW

File: src/plugins/cosmos/components/modals/UnstakingConfirm/UnstakingConfirm.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        <Flex minWidth={{ base: '100%', xl: '500px' }} flexDir={{ base: 'column', lg: 'row' }}>
+          <RouteSteps
+            assetSymbol={asset.symbol}
+            action={StakingAction.Unstake}
+            px={23}
+            py={43}
+            routes={withdrawRoutes}
+          />
+          <Flex
+            flexDir='column'
```
Comment (NeOMakinG) [inline src/plugins/cosmos/components/modals/UnstakingConfirm/UnstakingConfirm.tsx:59]:
```suggestion
            direction='column'
```

Comment (NeOMakinG) [reply src/plugins/cosmos/components/modals/UnstakingConfirm/UnstakingConfirm.tsx:59]: Just in order to be consistent

File: src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    <FormProvider {...methods}>
+      <AnimatePresence exitBeforeEnter initial={false}>
+        <Flex
+          as='form'
+          pt='14px'
+          pb='18px'
+          px='30px'
+          onSubmit={handleSubmit(onSubmit)}
+          direction='column'
+          alignItems='center'
```
Comment (NeOMakinG) [inline src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx:81]:
```suggestion
          align='center'
```

File: src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+          justifyContent='space-between'
```
Comment (NeOMakinG) [inline src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx:82]:
```suggestion
          justify='space-between'
```

Comment (NeOMakinG) [reply src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx:82]: There are others where shorthand are not used, feel free to change them too

Also: 

It needs to be defined, do we use shorthand everywhere, or do we use real properties everywhere?

Comment (reallybeard) [reply src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx:82]: I prefer we use the full name of the css property, justify/justifyContent, align/alignItems, etc...

I think this helps make it super clear that this is a flexbox element, vs a table element or something along those lines that also takes align props.

Comment (NeOMakinG) [reply src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx:82]: Yeah alright, but we should be consistent with this choice then :)

**Review by theoboldfrazier — APPROVED** [review]
Looks good

**Comment by gomesalexandre** [comment]
Rebasing after merge of #1140.

**Pull request merged**